### DISCUSSION TITLE
Different profile image and repository image

### DISCUSSION THREAD

**Discussion by SrAdryX** [community/community#44119] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, I have a bug or something strange with my github profile, let's say I have a profile image as you can see:
![Captura de pantalla 2023-01-12 165350](https://user-images.githubusercontent.com/75207206/212115378-6c3fe5ac-4966-4eef-9b7e-7ec1567ed93f.png)
So far so good, but when I upload a repository using the git terminal (git bash) the image that appears when entering the repository is different as you can see:
![Captura de pantalla 2023-01-12 165339](https://user-images.githubusercontent.com/75207206/212115391-858a8d0f-4a51-44a3-a9c0-0d0c76d1d55c.png)
I don't know where that image comes from, I just remember having used it in an email a long time ago but I don't use it anywhere anymore so I don't know where that incorrect profile image comes from, my question is, can it be changed? And if it can, how can I change it? I would appreciate all the help possible, I am new to github uploading files to repositories so I still get confused.
Is there a file where that image is and where I can change it or is there any command I can do to replace it with the profile image?
I am waiting for an answer, if someone knows how I can solve this, all the help is good for me.

**Comment by PetarRan** [comment]
Check what profile are you logged into on your system, your **git terminal (git bash)**.
I checked your repo right now and it appears its different profiles with similar credentials.

**Reply by SrAdryX** [reply]
Thanks for the help :)

**Comment by spenserblack** [comment, marked as answer]
You're using 2 different email addresses.
Some of your commits (the ones that show the correct profile) use `75207206+SrAdryX@users.noreply.github.com`. Others use `cib***@gmail.com` (I'm censoring in case you don't want me to share this, hopefully you know what I'm referring to). GitHub uses the email address to identify the user. You can see the email addresses using `git log`. You should try to add the email address to your GitHub account, or change the email address in `git config user.email`.

**Reply by SrAdryX** [reply]
Thank you very much, I was able to solve it as you have told me, it turns out that if I had another email configured that was not it, thank you very much, now it works fine for me :)
